integration-templates/hubspot/hubspot-service-tickets.ts

```diff
@@ -0,0 +1,97 @@
+import type { NangoSync, HubspotServiceTicket } from './models';
+
+export default async function fetchData(nango: NangoSync): Promise<{HubspotServiceTicket: HubspotServiceTicket[]}> {
+    const tickets = await paginate(nango, '/crm/v3/objects/tickets/search');
+
+    const mappedTickets: HubspotServiceTicket[] = tickets.map(ticket => ({
+        id: ticket.id,
+        createdAt: ticket.createdAt,
+        updatedAt: ticket.updatedAt,
+        archived: ticket.archived,
+        properties: {
+            content: ticket.properties.content,
+            createdate: ticket.properties.createdate,
+            hs_lastmodifieddate: ticket.properties.hs_lastmodifieddate,
```

Comment:
Curious here: why is there redundant create and updated dates in the properties since they exist at the top level?

Reply:
When you create a ticket in Hubspot Service it gives you the option to specify the "Created Date". I think the `createdate` in properties may be referring to that date. Not verified but I'll take a look to confirm